odoo/tools/translate.py

```diff
@@ -1570,7 +1570,10 @@ def _get_translation_upgrade_queries(cr, field):
               GROUP BY it.res_id
             )
             UPDATE {Model._table} m
-               SET "{field.name}" = CASE WHEN t.noupdate THEN m."{field.name}" || t.value ELSE t.value || m."{field.name}" END
+               SET "{field.name}" = CASE WHEN t.noupdate = true THEN m."{field.name}" || t.value
+                                         WHEN t.noupdate = false THEN t.value || m."{field.name}"
+                                         ELSE m."{field.name}" || t.value
+                                     END
```

Comment:
```suggestion
               SET "{field.name}" = CASE WHEN t.noupdate IS FALSE THEN t.value || m."{field.name}" ELSE m."{field.name}" || t.value END
```
`t.noupdate IS FALSE` will return `FALSE` when `t.noupdate IS NULL` 😃 



Could you provide a concrete example, why `t.noupdate IS NULL` should behave the same as `t.noupdate IS TRUE` for product_template and cowed website menu?
For example: 
what were the data before migration
what were the data before the end-migrate.py
what are the expected data after the end-migrate.py

Reply:
> `t.noupdate IS FALSE` will return `FALSE` when `t.noupdate IS NULL` smiley

@HydrionBurst Suggested Changes are done